src/services/TemplatingService.ts

```diff
@@ -0,0 +1,150 @@
+import { messaging as firebaseMessaging } from 'firebase-admin';
+import Mail from '@/interfaces/Mail';
+import Template from '../interfaces/Template';
+import TemplatePayload from '@/interfaces/TemplatePayload';
+import UserRessource from '@/interfaces/UserRessource';
+import Utils from '../utils';
+
+const MAIL_MESSAGE = 'MAIL';
+const PUSH_MESSAGE = 'PUSH';
+
+export default class TemplatingService {
```

Comment:
Please specify the types of all function parameters and the return type of each function explicitly using interfaces.

Reply:
I think we have to discuss how the templates will look in detail.

Reply:
Let's do this when implementing the templating service itself.